### DISCUSSION TITLE
Any way to automate RSS feeds refresh?

### DISCUSSION THREAD

**Discussion by greguti** [FreshRSS/FreshRSS#7087] [Q&A]
Hi! 

I read the page https://freshrss.github.io/FreshRSS/fr/users/03_Main_view.html#rafra%C3%AEchir-les-flux but I seem to understand that there is no way to automate refresh of all the RSS feeds, when FreshRSS is installed in a website (I mean, not in local)? 

My FreshRSS account is installed inside a specific folder of my personal website, so I can access it from anywhere. But I have to clic the "refresh" button each time? If so, it means that whatever Android app I use to access it (as an example, I use Readrops these days), I must first go to my FreshRSS in my website, clic the Refresh button, and then I can use my app with the updated feeds?

I think I miss something but I don't any "automatic update" function in the settings... ?

**Comment by Frenzie** [comment]
There are services like https://cron-job.org or GitHub Actions: https://www.theanshuman.dev/articles/free-cron-jobs-with-github-actions-31d6 you can use to trigger the relevant URL.

**Comment by math-GH** [comment]
Please take a look into the Engilsh documentation too: https://freshrss.github.io/FreshRSS/en/admins/08_FeedUpdates.html
